projects/js-packages/components/components/jetpack-footer/index.tsx

```diff
@@ -8,6 +8,10 @@ import JetpackLogo from '../jetpack-logo';
 import useBreakpointMatch from '../layout/use-breakpoint-match';
 import type { JetpackFooterProps } from './types';
 
+const JetpackIcon: React.FC = () => (
+	<JetpackLogo logoColor="#000" showText={ false } height={ 16 } aria-hidden="true" />
```

Comment:
Set `aria-hidden="true"` since the accessible name of the corresponding item is defined by `moduleName`.